openmetadata-ui/src/main/resources/ui/src/components/Glossary/ImportResult/ImportResult.tsx

```diff
@@ -0,0 +1,167 @@
+import { useTranslation } from 'react-i18next';
+import { parseCSV } from 'utils/GlossaryUtils';
+import { GlossaryCSVRecord } from '../ImportGlossary/ImportGlossary.interface';
+
+interface Props {
+  csvImportResult: CSVImportResult;
+}
+
+const ImportResult: FC<Props> = ({ csvImportResult }) => {
+  const { t } = useTranslation();
+
+  const parsedRecords: GlossaryCSVRecord[] = useMemo(() => {
+    const importResult = csvImportResult?.importResultsCsv;
+
+    if (importResult) {
+      return parseCSV(importResult);
+    } else {
+      return [];
+    }
+  }, [csvImportResult]);
+
+  const columns: ColumnsType<GlossaryCSVRecord> = useMemo(
+    () => [
+      {
+        title: t('label.status'),
+        dataIndex: 'status',
+        key: 'status',
+        render: (status: GlossaryCSVRecord['status']) => {
+          return (
+            <Typography.Text
+              className={classNames(
+                {
+                  'text-success': status === Status.Success,
+                },
+                { 'text-failure': status === Status.Failure }
+              )}>
+              {status}
+            </Typography.Text>
+          );
+        },
+      },
+      {
+        title: t('label.detail-plural'),
+        dataIndex: 'details',
+        key: 'details',
+        render: (details: GlossaryCSVRecord['details']) => {
+          return <Typography.Text>{details}</Typography.Text>;
+        },
+      },
+      {
+        title: t('label.parent'),
+        dataIndex: 'parent',
+        key: 'parent',
+        render: (parent: GlossaryCSVRecord['parent']) => {
+          return <Typography.Text>{parent}</Typography.Text>;
+        },
+      },
+      {
+        title: t('label.name'),
+        dataIndex: 'name*',
+        key: 'name',
+        render: (name: GlossaryCSVRecord['name*']) => {
+          return <Typography.Text>{name}</Typography.Text>;
+        },
+      },
+      {
+        title: t('label.display-name'),
+        dataIndex: 'displayName',
+        key: 'displayName',
+        render: (displayName: GlossaryCSVRecord['displayName']) => {
+          return <Typography.Text>{displayName}</Typography.Text>;
+        },
+      },
+      {
+        title: t('label.description'),
+        dataIndex: 'description*',
+        key: 'description',
+        render: (description: GlossaryCSVRecord['description*']) => {
+          return <RichTextEditorPreviewer markdown={description} />;
+        },
+      },
+      {
+        title: t('label.Synonyms'),
+        dataIndex: 'synonyms',
+        key: 'synonyms',
+        render: (synonyms: GlossaryCSVRecord['synonyms']) => {
+          return <Typography.Text>{synonyms}</Typography.Text>;
+        },
+      },
+      {
+        title: t('label.tag-plural'),
+        dataIndex: 'tags',
+        key: 'tags',
+        render: (tags: GlossaryCSVRecord['tags']) => {
+          return <Typography.Text>{tags}</Typography.Text>;
+        },
+      },
+    ],
+    [parsedRecords]
```

Comment:
Not sure why we have deps for parsedRecords for columns data

Reply:
We don't need that, I will remove it.